deployment/services/cloudflare-security.ts

```diff
@@ -51,14 +51,14 @@ export function deployCloudFlareSecurityTransform(options: {
 
   const contentSecurityPolicy = `
   default-src 'self';
-  frame-src ${stripeHost} https://game.crisp.chat https://giscus.app https://www.youtube.com;
```

Comment:
we use giscus on the docs 👀

Reply:
yeah but now it's no longer on the Hive app, because the landing page and docs are part of the main Guild website